### ISSUE TITLE
Validation of multiple root elements

### ISSUE THREAD

**Issue by bendrissou** [NaturalIntelligence/fast-xml-parser#617]
### Description
The parser accepts inputs that have multiple XML root nodes, without raising an error. This is despite enabling the validator option "-v". 

An XML document is expected to have one root element.

### Input
```xml
<abc/><abc/>
```
There are two elements at the root level, which I expected to raise an error. The input is saved in a file named `example.xml`.

### Code
```bash
./src/cli/cli.js -v example.xml
```

### Output
```
{ abc: [ '', '' ] }
```

My question is: Why the XML validator doesn't throw an error?
Please note that _xmllint_ rejects the input example as invalid.
